micropub.php

```diff
@@ -62,7 +62,7 @@ public static function init() {
     add_filter('host_meta', array('Micropub', 'jrd_links'));
     add_filter('webfinger_data', array('Micropub', 'jrd_links'));
 
-    // Store MF2
+    // Store MF2 - set at priority 8 in order to occur before the other filters.
```

Comment:
the valuable thing to comment here is *why* we want it to occur before the other filters. :P